tests/test-definition.yaml

```diff
@@ -31,6 +31,7 @@ tests:
     dimensions:
       - opa-latest
       - keycloak
+      - openshift
```

Comment:
LGTM, but I think we should comment out line 15, WDYT?

Reply:
Done. I didn't see it. Thanks.